elements/pf-progress-stepper/pf-progress-step.ts

```diff
@@ -13,9 +13,9 @@ import { InternalsController } from '@patternfly/pfe-core/controllers/internals-
 import style from './pf-progress-step.css';
 
 const ICONS = new Map(Object.entries({
-  success: { icon: 'circle-check' },
-  danger: { icon: 'circle-exclamation' },
-  warning: { icon: 'triangle-exclamation' },
+  success: { icon: 'check-circle' },
+  danger: { icon: 'exclamation-circle' },
+  warning: { icon: 'exclamation-triangle' },
```

Comment:
Because we are moving to the 'non-combined' set of icons which is purely font awesome v5, then these icons names no longer will work.